src/main.cpp

```diff
@@ -593,65 +596,31 @@ int main ( int argc, char** argv )
 
         if ( bUseGUI )
         {
-            if ( strDirectoryServer.isEmpty() )
-            {
-                // per definition: if we are in "GUI" server mode and no directory server
-                // address is given, we use the default directory server address
-                strDirectoryServer = DEFAULT_SERVER_ADDRESS;
-                qInfo() << qUtf8Printable ( QString ( "- default directory server set: %1" ).arg ( strDirectoryServer ) );
-            }
-        }
-        else
-        {
-            // the inifile is not supported for the headless server mode
-            if ( !strIniFileName.isEmpty() )
-            {
-                qWarning() << "No initialization file support in headless server mode.";
-            }
-        }
-
-        if ( strDirectoryServer.isEmpty() )
-        {
-            // per definition, we must be a headless server and ignoring inifile, so we have all the information
+            // by definition, when running with the GUI we always default to registering somewhere but
```

Comment:
This comment suggests that you can't have an unregistered ("private") server when running with the GUI. That's not true, surely (maybe it used to be?). Could the comment be clarified?

Reply:
No, it says what it defaults to, not what's available.